pkg/kubectl/explain/field_lookup.go

```diff
@@ -89,6 +89,12 @@ func (f *fieldLookup) VisitKind(k *proto.Kind) {
 	subSchema.Accept(f)
 }
 
+func (f *fieldLookup) VisitArbitrary(a *proto.Arbitrary) {
+	if f.SaveLeafSchema(a) {
```

Comment:
`f.Schema = a` would be more accurate (note that the condition is pointless).

Reply:
Make sense.